script/strategies/morpho/base/DeployStrategies.sol

```diff
@@ -7,20 +7,29 @@ import { DeployPeriphery } from "script/DeployPeriphery.sol";
 
 contract DeployStrategies is DeployPeriphery, BaseDeployStrategies {
   function run() external override(DeployPeriphery) {
+    address[] memory judges = new address[](1);
+    address msig = getMsig();
+    judges[0] = msig;
+
     vm.startBroadcast();
     deployPeriphery();
-    deployMorphoStrategies();
+    // Warning: Morpho Factory was updated, so this will deploy the strategies with the new factory
+    // TODO: Remove this when the new factory is deployed
+    _deployMorphoStrategies(getGuardianArrayWithMsig(BALMY_GUARDIAN), judges, BALMY_GUARDIAN_TOS_GROUP, "");
```

Comment:
Let's use named params, and should we comment this out for now based on the comment above?